src/platform/ESP32/BLEManagerImpl.h

```diff
@@ -205,6 +209,210 @@ inline bool BLEManagerImpl::_IsAdvertising(void)
 } // namespace DeviceLayer
 } // namespace chip
 
+#elif CONFIG_BT_NIMBLE_ENABLED
```

Comment:
Would it be possible to split the various ports into their own headers?

Reply:
Sorry if I was unclear in the previous commit. I was more thinking of having the same BLEManagerImpl definition with an #ifdef for the platform specific methods. See commit https://github.com/vivien-apple/connectedhomeip-1/commit/9c728e07003f3fc9a5d77259b529e4ada721012b that goes on top of your PR :)

Reply:
Hi @vivien-apple I have made the relevant changes !! Thanks.